### PR TITLE
[#469] Add copying all dependencies to EDGE image

### LINKED ISSUE #469: Move installation of all dependencies to EDGE dockerfile

### PR DESCRIPTION
**Pull request by kirillmakhonin**
This closes #469

### CODE REVIEW

**Comment by kirillmakhonin** [comment]
ok to test

File: k8s/edge/Dockerfile [hunk 1]
```diff
@@ -28,13 +28,10 @@ RUN set -ex \
...
-  && chmod +x /usr/bin/dumb-init \
-  && python3 -m pip install argparse jinja2 kubernetes urllib3 aionotify==0.2.0 \
-  && python3 -m pip install --upgrade pip==9.0.3 \
-  && python3 -m pip install "Pillow==5.0.0" \
-  && python3 -m pip install "numpy==1.14.3" \
-  && python3 -m pip install "pandas==0.23.0"
-# TODO: Remove Pillow, and pandas installation after toolchan system merge
+  && chmod +x /usr/bin/dumb-init
+
+COPY requirements.txt /root/requirements.txt
```
Comment (dsuslov) [inline k8s/edge/Dockerfile]: @kirillmakhonin please, add comments that this is temporal approach to decrease build time and the source of requirements.txt - legion/requirements/base.txt that is copied here during Jenkins pipeline execution (Jenkinsfile) This workaround should be removed as soon as #226 is done

**Comment by kirillmakhonin** [comment]
ok to test

**Comment by kirillmakhonin** [comment]
ok to test

**Pull request merged**